sdk/Lusid.Sdk.Tests/tutorials/Misc/ComplexInstruments.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Model;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Misc
+{
+    [TestFixture]
+    public class ComplexInstruments
+    {
+        [Test]
+        public void DemonstrateCreationOfFxForward()
+        {
+            // CREATE an Fx-Forward (that can then be upserted into LUSID)
+            var usdJpyFxRate = 109m; // Assume 1 USD is worth 109m as contract is struck. USD is domestic, JPY is foreign.
+            var fxForward = new FxForwardInstrument(
+                domAmount: 1m,
+                fgnAmount: usdJpyFxRate,
+                domCcy: "USD",
+                fgnCcy: "JPY",
+                startDate: new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero),
+                maturityDate: new DateTimeOffset(2020, 9, 18, 0, 0, 0, TimeSpan.Zero)
+                );
+
+            // ASSERT that it was created
+            Assert.That(fxForward, Is.Not.Null);
+            // CAN NOW UPSERT TO LUSID
```

Comment:
is there a reason we're not doing the upsert here (and below)?

Reply:
I didn't think anybody would need it. Have changed to have that.